paddle/fluid/framework/details/op_handle_graph.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+#include "paddle/fluid/framework/details/op_handle_base.h"
+
+namespace paddle {
+namespace framework {
+namespace details {
+
+class OpHandleGraph {
```

Comment:
A lot of methods in this class is not used by other modules. It seems that we could keep them in this class for further usage or we can keep this class clean until they are needed.

Reply:
`OpHandleGraph` is a confusing name. It just another view of IRGraph which only contains op.

Perhaps `OpGraphView` is better. `View` means this class does not change any data, just another view of graph. `OpGraph` means this graph only contains operators.

Reply:
For future use. Have deleted.